src/test/regress/sql/multi_extension.sql

```diff
@@ -468,11 +468,16 @@ ALTER EXTENSION citus UPDATE TO '11.0-1';
 ALTER EXTENSION citus UPDATE TO '11.0-2';
 SELECT * FROM multi_extension.print_extension_changes();
 
+-- Snapshot of state at 11.0-3
+ALTER EXTENSION citus UPDATE TO '11.0-3';
+SELECT * FROM multi_extension.print_extension_changes();
+
 -- Snapshot of state at 11.1-1
 ALTER EXTENSION citus UPDATE TO '11.1-1';
 SELECT * FROM multi_extension.print_extension_changes();
 
 -- Test downgrade script (result should be empty)
+ALTER EXTENSION citus UPDATE TO '11.0-3';
 ALTER EXTENSION citus UPDATE TO '11.0-2';
 ALTER EXTENSION citus UPDATE TO '11.1-1';
 SELECT * FROM multi_extension.print_extension_changes();
```

Comment:
We want to test downgrades from 11.1-1 down to 11.0-3 here. I think we should not need to test downgrades all the way back to 11.0-2 here as it should already be covered by a previous test.

```suggestion
-- Test downgrade script (result should be empty)
ALTER EXTENSION citus UPDATE TO '11.0-3';
ALTER EXTENSION citus UPDATE TO '11.1-1';
SELECT * FROM multi_extension.print_extension_changes();
```